homeassistant/components/venstar/sensor.py

```diff
@@ -0,0 +1,135 @@
+"""Representation of Venstar sensors."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import (
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_TEMPERATURE,
+    STATE_CLASS_MEASUREMENT,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE, TEMP_CELSIUS, TEMP_FAHRENHEIT
+from homeassistant.helpers.entity import Entity
+
+from . import VenstarDataUpdateCoordinator, VenstarEntity
+from .const import DOMAIN
+
+
+@dataclass
+class VenstarSensorEntityDescription(SensorEntityDescription):
+    """Base description of a Sensor entity."""
+
```

Comment:
You could extend this an add the `hum` and `temp` values as a mixin so you could loop though the sensors and add them without having to write them out in `async_setup_entry`

Example:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/august/binary_sensor.py#L94